### ISSUE TITLE
Treeview: VO doesn't announce collapsed and expanded states.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2420]
Using voiceover (and other screenreaders?) :

Move focus to collapsed top level item, and the it says the row is collapsed
Expand it, and it says the row is expanded
Move to a 2nd or 3rd level item and it says the row is collapsed
Expand it, and it says nothing. Should announce that you expanded it.

React Magma version: 4.12.1
Need to verify it with NVDA as well.

[Details](https://cengage.slack.com/archives/GCQLGU6Q6/p1775480941418789)
[Treeview + Checkbox](https://cengage.slack.com/archives/C8Y4FR63U/p1776334019631149)

**Comment by artemtrusov-cengage** [comment]
#### Tested on:

V5:  https://next--upbeat-sinoussi-f675aa.netlify.app/api/list/
Works as expected. (MacOS + VoiceOver / Windows + NVDA)

V4: https://rc--upbeat-sinoussi-f675aa.netlify.app/api/progress-bar/
Works as expected. (MacOS + VoiceOver / Windows + NVDA)
